app/dashboard/src/layouts/AssetsTable.tsx

```diff
@@ -2302,9 +2246,12 @@ export default function AssetsTable(props: AssetsTableProps) {
         {
           selectAdditional: () => {},
           selectAdditionalRange: () => {},
-          [inputBindingsModule.DEFAULT_HANDLER]: () => {
-            const { selectedKeys } = driveStore.getState()
-            if (selectedKeys.size !== 0) {
+          [inputBindingsModule.DEFAULT_HANDLER]: (event) => {
+            const portalRoot =
```

Comment:
What does this code do?

Reply:
added a comment. basically we look for `enso-portal-root` in the ancestors of the target to find out whether we clicked in a dialog (the datalink dialog), in which case we cancel deselecting the keys. otherwise we clicked outside the assets table and the keys should be deselected

Reply:
Weird. there should be a better solution.

Reply:
Though nothing is coming to mind. I'm ok with the current solution, but we need to find a less fragile one

Reply:
yup, agree

Reply:
opened as https://github.com/enso-org/cloud-v2/issues/1472